velox/functions/prestosql/aggregates/DecimalAggregate.h

```diff
@@ -38,8 +38,10 @@ struct LongDecimalWithOverflowState {
     overflow += stream.read<int64_t>();
     uint64_t lowerSum = stream.read<uint64_t>();
     int64_t upperSum = stream.read<int64_t>();
-    overflow += DecimalUtil::addWithOverflow(
-        this->sum, buildInt128(upperSum, lowerSum), this->sum);
+    int128_t lhsSum = buildInt128(upperSum, lowerSum);
+    int128_t rhsSum = this->getSum();
```

Comment:
No need to declare the variables. Use them directly in the addWithOverflow() call.

Reply:
Thanks. I have updated the code.